cmd/retrieve.go

```diff
@@ -0,0 +1,245 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"archive/zip"
+	"path/filepath"
+	"strings"
+
+	"github.com/skuid/skuid/pliny"
+	"github.com/skuid/skuid/types"
+	"github.com/spf13/cobra"
+)
+
+// retrieveCmd represents the retrieve command
+var retrieveCmd = &cobra.Command{
+	Use:   "retrieve",
+	Short: "Retrieve Skuid metadata from a Site into a local directory.",
+	Long: "Retrieve Skuid metadata from a Skuid Platform Site and output it into a local directory.",
+	Run: func(cmd *cobra.Command, args []string) {
+
+		if verbose {
+			fmt.Println("args")
+			fmt.Println(args)
+		}
+
+		//login to a Skuid Platform Site
+		api, err := pliny.Login(
```

Comment:
Oops - didn't realize this was public. I switched to using "platform" as the package name.